### ISSUE TITLE
Support for generating documentation via protoc-gen-doc

### ISSUE THREAD

**Issue by phiros** [os72/protoc-jar#39]
When protoc is called with the `--doc_out` flag it calls `protoc-gen-doc` in order to generate documentation for supplied protobuf files. This fails on systems where `protoc-gen-doc` is not installed. Therefore, it would be really useful (and convenient) if protoc-jar would come bundled with this binary.

**Comment by os72** [comment]
I don't think we would start bundling plugins, but we could make them auto-downloadable (from maven central) just like the main `protoc` binary

How does your `protoc-jar` invocation currently look (cmd line)?
